<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DB_PaletteColorPickerWidget</class>
 <widget class="QWidget" name="DB_PaletteColorPickerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QPushButton" name="pbDefault">
     <property name="toolTip">
      <string>Set to default color</string>
     </property>
     <property name="text">
      <string>&amp;Default</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="pbReverse">
     <property name="minimumSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Reverse palette color using Red Green Blue setting</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="db.qrc">
       <normaloff>:/images/icons/arrow_switch.png</normaloff>:/images/icons/arrow_switch.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pbBase">
     <property name="toolTip">
      <string>Palette color using Red Green Blue setting</string>
     </property>
     <property name="text">
      <string>&amp;Base</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QSpinBox" name="sbBlue">
     <property name="prefix">
      <string>Blue: </string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>64</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="sbRed">
     <property name="prefix">
      <string>Red: </string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>64</number>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QPushButton" name="pbPreview">
     <property name="minimumSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Show colors in preview</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="db.qrc">
       <normaloff>:/images/icons/magnifier.png</normaloff>:/images/icons/magnifier.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="9" rowspan="2">
    <widget class="QTabWidget" name="tabWidgetPreview">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabActive">
      <attribute name="title">
       <string>Active</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>-434</y>
            <width>354</width>
            <height>1117</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0" colspan="2">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" rowspan="2">
            <widget class="QCommandLinkButton" name="commandLinkButton">
             <property name="text">
              <string>CommandLinkButton</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QToolButton" name="toolButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QRadioButton" name="radioButton">
             <property name="text">
              <string>RadioButton</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QDialogButtonBox" name="buttonBox">
             <property name="standardButtons">
              <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QRadioButton" name="radioButton_2">
             <property name="text">
              <string>RadioButton</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2" rowspan="11">
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>GroupBox</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <property name="leftMargin">
               <number>3</number>
              </property>
              <property name="topMargin">
               <number>3</number>
              </property>
              <property name="rightMargin">
               <number>3</number>
              </property>
              <property name="bottomMargin">
               <number>3</number>
              </property>
              <property name="spacing">
               <number>3</number>
              </property>
              <item row="0" column="0">
               <widget class="QListWidget" name="listWidget">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QTreeWidget" name="treeWidget">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <column>
                 <property name="text">
                  <string notr="true">1</string>
                 </property>
                </column>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QTableWidget" name="tableWidget">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>CheckBox</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QComboBox" name="comboBox"/>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QFontComboBox" name="fontComboBox"/>
           </item>
           <item row="7" column="0" colspan="2">
            <widget class="QLineEdit" name="lineEdit">
             <property name="text">
              <string>Some text</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0" colspan="2">
            <widget class="QTextEdit" name="textEdit"/>
           </item>
           <item row="9" column="0" colspan="2">
            <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
           </item>
           <item row="10" column="0" colspan="2">
            <widget class="QDateTimeEdit" name="dateTimeEdit"/>
           </item>
           <item row="11" column="0" colspan="2">
            <widget class="QSlider" name="horizontalSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="12" column="0" colspan="2">
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="13" column="0" rowspan="3">
            <widget class="QSlider" name="verticalSlider">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="13" column="1" rowspan="2">
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="14" column="2">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Below is a graphics view</string>
             </property>
            </widget>
           </item>
           <item row="15" column="2" rowspan="4">
            <widget class="QGraphicsView" name="graphicsView"/>
           </item>
           <item row="16" column="0" colspan="2">
            <widget class="QLCDNumber" name="lcdNumber">
             <property name="digitCount">
              <number>5</number>
             </property>
             <property name="mode">
              <enum>QLCDNumber::Dec</enum>
             </property>
             <property name="value" stdset="0">
              <double>2014.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="17" column="0" colspan="2">
            <widget class="QProgressBar" name="progressBar">
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
           <item row="19" column="0" colspan="3">
            <widget class="QCalendarWidget" name="calendarWidget">
             <property name="gridVisible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDisabled">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Disabled</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>354</width>
            <height>1117</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0" colspan="2">
            <widget class="QPushButton" name="pushButton_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" rowspan="2">
            <widget class="QCommandLinkButton" name="commandLinkButton_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>CommandLinkButton</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QToolButton" name="toolButton_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QRadioButton" name="radioButton_3">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>RadioButton</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QDialogButtonBox" name="buttonBox_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="standardButtons">
              <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QRadioButton" name="radioButton_4">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>RadioButton</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2" rowspan="11">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="title">
              <string>GroupBox</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <property name="leftMargin">
               <number>3</number>
              </property>
              <property name="topMargin">
               <number>3</number>
              </property>
              <property name="rightMargin">
               <number>3</number>
              </property>
              <property name="bottomMargin">
               <number>3</number>
              </property>
              <property name="spacing">
               <number>3</number>
              </property>
              <item row="0" column="0">
               <widget class="QListWidget" name="listWidget_2">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QTreeWidget" name="treeWidget_2">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <column>
                 <property name="text">
                  <string notr="true">1</string>
                 </property>
                </column>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QTableWidget" name="tableWidget_2">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QCheckBox" name="checkBox_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>CheckBox</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QComboBox" name="comboBox_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QFontComboBox" name="fontComboBox_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="7" column="0" colspan="2">
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Some text</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0" colspan="2">
            <widget class="QTextEdit" name="textEdit_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="9" column="0" colspan="2">
            <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="10" column="0" colspan="2">
            <widget class="QDateTimeEdit" name="dateTimeEdit_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="11" column="0" colspan="2">
            <widget class="QSlider" name="horizontalSlider_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="12" column="0" colspan="2">
            <widget class="Line" name="line_3">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="13" column="0" rowspan="3">
            <widget class="QSlider" name="verticalSlider_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="13" column="1" rowspan="2">
            <widget class="Line" name="line_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="14" column="2">
            <widget class="QLabel" name="label_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Below is a graphics view</string>
             </property>
            </widget>
           </item>
           <item row="15" column="2" rowspan="4">
            <widget class="QGraphicsView" name="graphicsView_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="16" column="0" colspan="2">
            <widget class="QLCDNumber" name="lcdNumber_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="digitCount">
              <number>5</number>
             </property>
             <property name="mode">
              <enum>QLCDNumber::Dec</enum>
             </property>
             <property name="value" stdset="0">
              <double>2014.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="17" column="0" colspan="2">
            <widget class="QProgressBar" name="progressBar_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
           <item row="19" column="0" colspan="3">
            <widget class="QCalendarWidget" name="calendarWidget_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="gridVisible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QSpinBox" name="sbGreen">
     <property name="prefix">
      <string>Green: </string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>64</number>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QPushButton" name="pbApply">
     <property name="minimumSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Apply to application</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="db.qrc">
       <normaloff>:/images/icons/bullet_go.png</normaloff>:/images/icons/bullet_go.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="9">
    <widget class="QTableWidget" name="twColor">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="toolTip">
      <string>Use below color in palette</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pbDefault</tabstop>
  <tabstop>pbBase</tabstop>
  <tabstop>sbRed</tabstop>
  <tabstop>sbGreen</tabstop>
  <tabstop>sbBlue</tabstop>
  <tabstop>pbReverse</tabstop>
  <tabstop>pbPreview</tabstop>
  <tabstop>pbApply</tabstop>
  <tabstop>twColor</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>radioButton</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>commandLinkButton</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>treeWidget</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>fontComboBox</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>doubleSpinBox</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>horizontalSlider</tabstop>
  <tabstop>verticalSlider</tabstop>
  <tabstop>graphicsView</tabstop>
  <tabstop>calendarWidget</tabstop>
  <tabstop>tabWidgetPreview</tabstop>
  <tabstop>scrollArea_2</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>commandLinkButton_2</tabstop>
  <tabstop>toolButton_2</tabstop>
  <tabstop>radioButton_3</tabstop>
  <tabstop>radioButton_4</tabstop>
  <tabstop>listWidget_2</tabstop>
  <tabstop>treeWidget_2</tabstop>
  <tabstop>tableWidget_2</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>fontComboBox_2</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>textEdit_2</tabstop>
  <tabstop>doubleSpinBox_2</tabstop>
  <tabstop>dateTimeEdit_2</tabstop>
  <tabstop>horizontalSlider_2</tabstop>
  <tabstop>verticalSlider_2</tabstop>
  <tabstop>graphicsView_2</tabstop>
  <tabstop>calendarWidget_2</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="db.qrc"/>
 </resources>
 <connections/>
</ui>
